lib/creek/sheet.rb

```diff
@@ -99,6 +99,10 @@ def rows_generator include_meta_data=false, use_simple_rows_format=false
           cell_style_idx = nil
           @book.files.file.open(path) do |xml|
             prefix = ''
+            name_row = "row"
+            name_c = "c"
+            name_v = "v"
+            name_t = "t"
             Nokogiri::XML::Reader.from_io(xml).each do |node|
               if prefix.empty? && node.namespaces.any?
                 namespace = node.namespaces.detect{|_key, uri| uri == SPREADSHEETML_URI }
```

Comment:
In the case that `prefix` has not been set and there are namespaces that are not `SPREADSHEETML_URI`, the code is still going to repeatedly set the `name_*` variables.

Perhaps this would make sense?

```suggestion
              if prefix.empty? && (namespace = node.namespaces.detect{|_key, uri| uri == SPREADSHEETML_URI }).present?
```

Reply:
I guess that suggestion leaves some quirks around the `if namespace && ...` check below...

Reply:
> In the case that `prefix` has not been set and there are namespaces that are not `SPREADSHEETML_URI`, the code is still going to repeatedly set the `name_*` variables.
> 
> Perhaps this would make sense?

I don't know if this is a common usecase.

If it is, we need to track prefix changes in order to handle new prefix and prefix reset :

```RUBY
if prefix != old_prefix
  name_row = "#{prefix}row"
  name_c = "#{prefix}c"
  name_v = "#{prefix}v"
  name_t = "#{prefix}t"
end
```